test/TestDSP.c

```diff
@@ -102,6 +112,14 @@ void test_dsp_object(void)
     TEST_ASSERT_DOUBLE_WITHIN(0.01, -16.653, dspv5_object_helper(-5000, -5000, 22454, 23030));
 }
 
+void test_dsp_object_reflected(void)
+{
+    TEST_ASSERT_DOUBLE_WITHIN(0.01, 55.507, dspv5_object_reflected_helper(609, 611, 22454, 23030, 40.00));
```

Comment:
Here we could also test that emissivity 1 with ambient temperature same as reflected tempearature, return the same value.